vector/src/main/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewFragment.kt

```diff
@@ -105,7 +108,7 @@ class SessionOverviewFragment :
         updateToolbar(state.isCurrentSession)
         updateEntryDetails(state.deviceId)
         if (state.deviceInfo is Success) {
-            renderSessionInfo(state.isCurrentSession, state.deviceInfo.invoke())
+            renderSessionInfo(state.isCurrentSession, state.deviceInfo.invoke(), state.isCurrentSessionTrusted)
```

Comment:
Nit: Maybe passing the state as a single argument could be simpler, but maybe not, because of the type check of `state.deviceInfo`

Reply:
I moved all the logic into a `updateSessionInfo()` method. See [f54955d](https://github.com/vector-im/element-android/pull/7156/commits/f54955db1d98269fb27ae6237baa546e8daec1b0)